src/sentry/auth/superuser.py

```diff
@@ -275,12 +292,54 @@ def set_logged_in(self, user, current_datetime=None):
         request = self.request
         if current_datetime is None:
             current_datetime = timezone.now()
+
+        token = get_random_string(12)
+
+        if is_self_hosted() or not self._need_validation():
+            self._set_logged_in(
+                expires=current_datetime + MAX_AGE,
+                token=token,
+                user=user,
+                current_datetime=current_datetime,
+            )
+
+            logger.info(
+                "superuser.logged-in",
+                extra={"ip_address": request.META["REMOTE_ADDR"], "user_id": user.id},
+            )
+            return
+
+        try:
+            # need to use json loads as the data is no longer in request.data
+            su_access_json = json.loads(request.body)
+        except AttributeError:
+            su_access_json = []
```

Comment:
nit: if the `su_access_json` above is to be a dict once loaded, its also a good idea to make this a dict

Reply:
yup, i realized this in the [following pr](https://github.com/getsentry/sentry/blob/18459105cdbe444582d39a9065aed163f86ce5ee/src/sentry/auth/superuser.py#L316), ill make the change here as well